server/superdesk/publish/odbc_test.py

```diff
@@ -15,7 +17,7 @@
 
 
 class ODBCTests(TestCase):
-    subscribers = [{"_id": "1", "name": "Test", "can_send_takes_packages": False, "media_type": "media",
+    subscribers = [{"_id": "1", "name": "Test", "subscriber_type": SUBSCRIBER_TYPES.WIRE, "media_type": "media",
```

Comment:
@akintolga I checked with Andrew on this. He is not sure if Media Type becomes redundant after introducing Subscriber Type. He wants you to confirm on this part. If you say Media Type is redundant then I will remove the property from both Frondend and Backend.